addons/mail/static/src/components/message_tracking_value/message_tracking_value.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="mail.MessageTrackingValue" owl="1">
+        <t t-if="messageTrackingValue">
+            <div class="o_MessageTrackingValue d-inline-block" t-attf-class="{{ className }}" t-ref="root">
+                <span class="o_MessageTrackingValue_changedField o_MessageTrackingValue_item" t-esc="messageTrackingValue.changedFieldAsString"/>
+                <t t-if="messageTrackingValue.oldValue.formattedValue">
+                    <span class="o_MessageTrackingValue_oldValue o_MessageTrackingValue_item" t-esc="messageTrackingValue.oldValue.formattedValue"/>
+                </t>
+                <i t-else="" class="mr-2">None</i>
```

Comment:
I think `formattedValue` should be set to `"None"` in its compute, allowing you to dump the value unconditionnaly here.

Reply:
Agree. Doing so likely would need to adapt template and introduce some other fields. So that:
- `<i>` becomes a `font-italic` conditional classname on "None" value

Make sure that you properly define what's a "None" value in modelling. The string "None" is just UI, and it shouldn't be mislead with actual value "None" (that's why there's the subtle style for this specific value).